config/src/main/java/org/springframework/security/config/authentication/AuthenticationManagerFactoryBean.java

```diff
@@ -61,13 +62,13 @@ public AuthenticationManager getObject() throws Exception {
 			if (!BeanIds.AUTHENTICATION_MANAGER.equals(ex.getBeanName())) {
 				throw ex;
 			}
-			UserDetailsService uds = getBeanOrNull(UserDetailsService.class);
+			UserDetailsService uds = this.bf.getBeanProvider(UserDetailsService.class).getIfUnique();
```

Comment:
Same comment here about `getIfAvailable`

Reply:
It's also same `getIfUnique` comment